src/views/Header/Header.js

```diff
@@ -123,14 +123,15 @@ class Header extends React.PureComponent<Props, State> {
           }
         : undefined;
 
+    const UsedHeaderTitle = details.options.HeaderTitleComponent || HeaderTitle;
     return (
-      <HeaderTitle
+      <UsedHeaderTitle
```

Comment:
Not a fan of `Used` for naming. Perhaps `RenderedHeaderTitle`?

Reply:
I agree